### DISCUSSION TITLE
[IntelliSense] The 'Tailwind CSS IntelliSense' extension is not available in Visual Studio Code for the Web.

### DISCUSSION THREAD

**Discussion by patotoma** [tailwindlabs/tailwindcss#13671] [Ideas]
<img width="854" alt="Screenshot 2024-05-12 at 8 57 50 AM" src="https://github.com/tailwindlabs/tailwindcss/assets/3104587/4efef897-5e96-43e2-aeb4-075f5464516f">

Wondering why is that and if it's going to be eventually resolved?

Thanks a lot.

EDIT: also it's refusing to work with virtual workspaces

**Comment by fabionebbia** [comment]
Stumbled on the same problem with virtual workspaces just now.

Any news about this problem?
